src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -121,11 +127,11 @@ public Task Initialize(TimeSpan timeout)
             try
             {
                 var watch = Stopwatch.StartNew();
-                messages = (await receiver.ReceiveAsync(maxCount, ReceiveTimeout)).ToList();
+                messages = (await receiver.ReceiveAsync(maxCount, ReceiveTimeout))?.ToList();
```

Comment:
Yes, I caugh null reference exceptions while porting and found out that yes, it can now return null. Here is the source https://github.com/Azure/azure-event-hubs-dotnet/blob/8774bc4afbfbe44f1722ee648cbaca7ae07edede/src/Microsoft.Azure.EventHubs/Amqp/AmqpPartitionReceiver.cs#L119